Rakefile

```diff
@@ -36,4 +36,22 @@ Rake::TestTask.new do |t|
   t.verbose = true
 end
 
-task default: [:test, :rubocop]
+task default: [:test, 'rubocop:auto_correct']
+desc 'CI test task'
+task :ci => [:test, 'ci:rubocop']
+namespace :ci do
+  require 'rbconfig'
+  # https://github.com/bundler/bundler/blob/1b3eb2465a64d938490110e84b5ca97a51253d14/lib/bundler/constants.rb#L2
+  windows_platforms = /(msdos|mswin|djgpp|mingw)/
+  if RbConfig::CONFIG['host_os'] !~ windows_platforms
```

Comment:
Do you think it is possible to move this Windows check to the before conditional  https://github.com/rails-api/active_model_serializers/pull/1224/files#diff-52c976fc38ed2b4e3b1192f8a8e24cffR16 ? I think this task is related to windows at all and not CI Windows only. Makes sense?